### ISSUE TITLE
Print stylesheet unexpectedly gets `onload` attribute

### ISSUE THREAD

**Issue by swissspidy** [addyosmani/critical#552]
If I understand #444 / 07be40bab23eb518bb689062b43f0db507c9a1f9 correctly, print stylesheets should be unaffected by the script. And that seems to actually be the case—the print stylesheet is not part of the extracted critical CSS.

_However_, the `<link>` tag still erroneously gets an `onload` attribute, which does not make sense.

Input HTML:

```html
<link rel="stylesheet" href="css/print.css" media="print">
```

Expected output:

(unchanged)

Actual output:

```html
<link rel="stylesheet" href="css/print.css" media="print" onload="this.media='print'">
````

**Comment by bezoerb** [comment]
Thanks for the issue 😊
I'll try to find some time to look into this in the next days.

**Comment by bezoerb** [comment]
Moved this issue to inline-critical because it's responsible for the inlining & rewriting.
Keeping this open until the issue is fixed upstream.

**Comment by bezoerb** [comment]
Should be fixed in [v6.0.2](https://github.com/addyosmani/critical/releases/tag/v6.0.2)
